app/lib/submission_builder/ty2022/states/ny/individual_return.rb

```diff
@@ -166,7 +166,10 @@ def supported_documents
             calculated_fields = tax_calculator.calculate
             receiving_213_credit = calculated_fields[:IT213_LINE_14].present? && calculated_fields[:IT213_LINE_14] > 0 && !@submission.data_source.direct_file_data.claimed_as_dependent?
             receiving_214_credit = calculated_fields[:IT214_LINE_33].present? && calculated_fields[:IT214_LINE_33] > 0
-            receiving_215_credit = calculated_fields[:IT215_LINE_1].present? && !calculated_fields[:IT215_LINE_2]
+            receiving_215_credit = calculated_fields[:IT215_LINE_1].present? &&
+                                   !calculated_fields[:IT215_LINE_2] &&
+                                    calculated_fields[:IT215_LINE_16] > 0 &&
+                                    calculated_fields[:IT215_LINE_27] > 0
```

Comment:
I think this may be wrong - Acceptance criteria reads: `If both E_EITC_CR_AMT **and** E_NYC_EITC_CR_AMT are $0 or less than $0`

So given that we are negating the criteria, I think the AND becomes an OR. e.g.:
```
            has_eitc_credit = calculated_fields[:IT215_LINE_16] > 0 || calculated_fields[:IT215_LINE_27] > 0
            receiving_215_credit = calculated_fields[:IT215_LINE_1].present? &&
                                   !calculated_fields[:IT215_LINE_2] &&
                                   has_eitc_credit
```
Though I'd probably break it up more for clarity.

Reply:
Hmm good catch

Reply:
Done